.drone/drone.yml

```diff
@@ -878,6 +878,40 @@ trigger:
 type: docker
 ---
 kind: pipeline
+name: Build agent-boringcrypto (Linux amd64 boringcrypto)
```

Comment:
nit: consider moving under `Build agent-flow (FreeBSD amd64)` so all the agent builds are grouped together

Reply:
ah right, we have the jsonnet generating this file. no worries it was a small nit. not worth spending much time on.